scripts/deployment/l2/arbitrum/timelock_token_mint.js

```diff
@@ -0,0 +1,123 @@
+/*global process*/
+
+const { ethers } = require("hardhat");
+const { L1ToL2MessageGasEstimator } = require("@arbitrum/sdk/dist/lib/message/L1ToL2MessageGasEstimator");
+const { EthBridger, getL2Network } = require("@arbitrum/sdk");
+const { getBaseFee } = require("@arbitrum/sdk/dist/lib/utils/lib");
+
+const main = async () => {
+    // Setting up providers and wallets
+    const ALCHEMY_API_KEY_SEPOLIA = process.env.ALCHEMY_API_KEY_SEPOLIA;
+    const sepoliaURL = "https://eth-sepolia.g.alchemy.com/v2/" + ALCHEMY_API_KEY_SEPOLIA;
+    const sepoliaProvider = new ethers.providers.JsonRpcProvider(sepoliaURL);
+    await sepoliaProvider.getBlockNumber().then((result) => {
+        console.log("Current block number sepolia: " + result);
+    });
+
+    const arbitrumSepoliaURL = "https://sepolia-rollup.arbitrum.io/rpc";
+    const arbitrumSepoliaProvider = new ethers.providers.JsonRpcProvider(arbitrumSepoliaURL);
+    await arbitrumSepoliaProvider.getBlockNumber().then((result) => {
+        console.log("Current block number arbitrum sepolia: " + result);
+    });
+
+    // Get the EOA
+    const account = ethers.utils.HDNode.fromMnemonic(process.env.TESTNET_MNEMONIC).derivePath("m/44'/60'/0'/0/0");
+    const EOAsepolia = new ethers.Wallet(account, sepoliaProvider);
+    const EOAarbitrumSepolia = new ethers.Wallet(account, arbitrumSepoliaProvider);
+    console.log("EOA", EOAsepolia.address);
+    if (EOAarbitrumSepolia.address == EOAsepolia.address) {
+        console.log("Correct wallet setup");
+    }
+
+    const mockTimelockAddress = "0x31D3202d8744B16A120117A053459DDFAE93c855";
+    const mockTimelock = (await ethers.getContractAt("MockTimelock", mockTimelockAddress)).connect(EOAsepolia);
+    //console.log(mockTimelock.address);
+
+    const tokenAddress = "0xeDd71796B90eaCc56B074C39BAC90ED2Ca6D93Ee";
+    const erc20Token = (await ethers.getContractAt("ERC20Token", tokenAddress)).connect(EOAarbitrumSepolia);
+    //console.log(erc20Token.address);
+
+    // Use l2Network to create an Arbitrum SDK EthBridger instance
+    // We'll use EthBridger to retrieve the Inbox address
+    const l2Network = await getL2Network(arbitrumSepoliaProvider);
+    const ethBridger = new EthBridger(l2Network);
+    const inboxAddress = ethBridger.l2Network.ethBridge.inbox;
+    console.log(inboxAddress);
+
+    // Query the required gas params using the estimateAll method in Arbitrum SDK
+    const l1ToL2MessageGasEstimate = new L1ToL2MessageGasEstimator(arbitrumSepoliaProvider);
+    //console.log(l1ToL2MessageGasEstimate);
+
+    // To be able to estimate the gas related params to our L1-L2 message, we need to know how many bytes of calldata out
+    // retryable ticket will require
+    const calldata = erc20Token.interface.encodeFunctionData("transfer", [EOAarbitrumSepolia.address, 100]);
```

Comment:
This script transfers tokens on L2 side from timelock alias, assuming it has funds (for example, from draining).